### DISCUSSION TITLE
Chlorophyll a plots from PACE data.

### DISCUSSION THREAD

**Discussion by Philliec459** [opengeos/HyperCoast#32] [Ideas]
Is it possible for us to plot Chlorophyll a from our PACE data in HyperCoast? The attached image is from NASA WorldView showing such a plot as an example. 

<img width="963" alt="WorldView_PACE" src="https://github.com/opengeos/HyperCoast/assets/56929340/8988f16c-d610-4d8c-9117-70ff46718485">

**Comment by bingqing-liu** [comment]
Thank you Craig for reaching out! Yes, we will add it soon!


Bingqing Liu

Assistant Professor in Environmental Science

Mid-Winter Fair Endowed Professorship of Renewable Resources


School of Geosciences

University of Louisiana at Lafayette

611 McKinley St., Hamilton Hall 335

Lafayette, LA 70504, USA

Mobile: +1 225-916-8089

UL Lafayette Marine Optics Lab
<https://sites.google.com/view/marineopticslab>

**Reply by Philliec459** [reply]
We have made an attempt to calculate chlorophyll a using HyperCoast. It can be found in our Jupyter Notebook on GitHub:

https://github.com/Philliec459/Downloading-of-NASA-PACE-Image-data-using-our-STS-Customized-HyperCoast-Software

Is there any way that we could plot this using your HyperCoast mapping code? Right now we are using matplotlib, but HyperCoast mapping is extremely good.
